### DISCUSSION TITLE
not working on localhost/project/admin without port

### DISCUSSION THREAD

**Discussion by codeqasim** [filamentphp/filament#2360] [Help]
### Package

filament/filament

### Package Version

2

### Laravel Version

9

### Livewire Version

2

### PHP Version

8

### Bug description

There is problem with the script it does not work on direct path without port 
We are trying to build project and most of our web servers run without port we want to host the script using cpanel files 

Please let us know how can we resolve this issue. the UI works but we cant login  

### Relevant log output

```shell
screenshot https://i.imgur.com/jhic5hq.png
```

**Comment by danharrin** [comment]
Check your APP_URL is correct, and that your web server is permitted to return .css and .js files from PHP

**Comment by danharrin** [comment]
This is a Livewire issue and not Filament, so I'm transferring this to a discussion

**Comment by codeqasim** [comment]
APP_URL is correct can you please explain what do you mean by return js and css by PHP ? 
by the way I updated my APP_URL and still its same issue
